main.nf

```diff
@@ -1,14 +1,179 @@
 #!/usr/bin/env nextflow
 
 nextflow.enable.dsl=2
+idList = []     // Global variable to save IDs from samplesheet
 
 include { CALL_STAPHYLOCOCCUS_AUREUS        } from './workflows/staphylococcus_aureus.nf'
 include { CALL_MYCOBACTERIUM_TUBERCULOSIS   } from './workflows/mycobacterium_tuberculosis.nf'
 
+<<<<<<< HEAD
```

Comment:
Contains bad merge

Reply:
Yeah this happens when you merge and have conflicts that you need to resolve. These lines should be deleted.